app/src/main/java/com/pysiak/simple/clock/ui/alarm/AlarmFragment.java

```diff
@@ -0,0 +1,31 @@
+package com.pysiak.simple.clock.ui.alarm;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.pysiak.simple.clock.R;
+
+/**
+ * Alarm Fragment implementation {@link AlarmView}
+ *
+ * @author Yaroslav P.
+ * {@link Fragment}
```

Comment:
Remove {@link Fragment}
Don't use it for standart classes from Android/Java Api